package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "7.1.11",
+  "version": "7.2.0",
```

Comment:
Was thinking this is a minor upgrade since it partially includes the legal name fix. I'm open to your thoughts if you think otherwise (7.1.12). Let me know!

Reply:
Minor upgrade sounds good.